src/ucp/rndv/rndv.c

```diff
@@ -246,26 +248,26 @@ ucp_rndv_reg_send_buffer(ucp_request_t *sreq, const ucp_request_param_t *param)
     ucp_md_map_t md_map;
     ucs_status_t status;
 
-    if (UCP_DT_IS_CONTIG(sreq->send.datatype) &&
-        ucp_rndv_is_get_zcopy(sreq, ep->worker->context)) {
-
-        /* register a contiguous buffer for rma_get */
-        md_map = ucp_ep_config(ep)->key.rma_bw_md_map;
+    if (UCP_DT_IS_CONTIG(sreq->send.datatype)) {
+        md_map = ucp_ep_config(ep)->key.rma_bw_md_map |
+                 UCS_BIT(ucp_ep_md_index(ep, ucp_ep_get_am_lane(ep)));
```

Comment:
can remove it since we plan to replace uct_memh in dt_reg by ucp_memh. for now, rndv am_zcopy will not be optimal.
cc @Artemy-Mellanox